internal/services/api/api_service.go

```diff
@@ -166,6 +166,10 @@ func RouterHandlerFunc(base util.AbsolutePath, lister accounts.AccountList, log
 	r.Handle(ToPath("deployments", "{name}"), DeleteDeploymentHandlerFunc(base, log)).
 		Methods(http.MethodDelete)
 
+	// GET /api/deployments/$NAME/environment
+	r.Handle(ToPath("deployments", "{name}", "environment"), GetDeploymentEnvironmentHandlerFunc(base, log, lister)).
```

Comment:
Unsure about the name here. Maybe we should be more direct about this being the Connect Content environment?

This is on the `deployment` resource which implies this is the content, and not the configuration's environment.

Reply:
I think it makes sense as you have it 👍